lib/pure/math.nim

```diff
@@ -163,6 +163,24 @@ proc prod*[T](x: openArray[T]): T {.noSideEffect.} =
   result = 1.T
   for i in items(x): result = result * i
 
+proc cumsummed*[T](x: openArray[T]): seq[T] =
+  ## Return cumulative aka prefix summation of ``x``.
+  ##
+  ## .. code-block:: nim
+  ##   var x = [1, 2, 3, 4]
+  ##   echo x.cumsum    # [1, 3, 6, 10]
```

Comment:
The example needs to be updated to `cumsummed`.

(Btw, would this have been caught if this was `runnableExample`?)

Reply:
Fixed.